[2025-04-21T21:15:36.977+0000] {processor.py:186} INFO - Started process (PID=169) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:15:36.978+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:15:36.981+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:15:36.981+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:15:36.989+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:15:36.986+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:15:36.991+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:15:37.012+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.052 seconds
[2025-04-21T21:16:07.571+0000] {processor.py:186} INFO - Started process (PID=186) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:16:07.573+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:16:07.583+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:16:07.582+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:16:07.589+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:16:07.588+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:16:07.589+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:16:07.616+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.054 seconds
[2025-04-21T21:16:38.028+0000] {processor.py:186} INFO - Started process (PID=206) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:16:38.029+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:16:38.033+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:16:38.032+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:16:38.038+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:16:38.037+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:16:38.038+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:16:38.063+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.045 seconds
[2025-04-21T21:17:08.381+0000] {processor.py:186} INFO - Started process (PID=226) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:17:08.382+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:17:08.386+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:17:08.385+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:17:08.392+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:17:08.390+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:17:08.393+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:17:08.418+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.049 seconds
[2025-04-21T21:17:39.395+0000] {processor.py:186} INFO - Started process (PID=254) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:17:39.397+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:17:39.402+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:17:39.402+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:17:39.409+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:17:39.408+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:17:39.410+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:17:39.424+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.042 seconds
[2025-04-21T21:18:09.648+0000] {processor.py:186} INFO - Started process (PID=274) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:18:09.649+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:18:09.653+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:18:09.652+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:18:09.660+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:18:09.658+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:18:09.661+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:18:09.687+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.050 seconds
[2025-04-21T21:18:39.874+0000] {processor.py:186} INFO - Started process (PID=294) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:18:39.874+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:18:39.877+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:18:39.877+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:18:39.883+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:18:39.881+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:18:39.883+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:18:39.908+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.044 seconds
[2025-04-21T21:19:10.029+0000] {processor.py:186} INFO - Started process (PID=314) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:19:10.030+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:19:10.034+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:19:10.033+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:19:10.041+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:19:10.039+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:19:10.041+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:19:10.064+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.054 seconds
[2025-04-21T21:19:40.370+0000] {processor.py:186} INFO - Started process (PID=334) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:19:40.371+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:19:40.375+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:19:40.375+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:19:40.383+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:19:40.381+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:19:40.384+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:19:40.413+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.055 seconds
[2025-04-21T21:20:10.620+0000] {processor.py:186} INFO - Started process (PID=354) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:20:10.621+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:20:10.626+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:20:10.625+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:20:10.632+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:20:10.631+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:20:10.632+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:20:10.659+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.050 seconds
[2025-04-21T21:20:40.935+0000] {processor.py:186} INFO - Started process (PID=374) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:20:40.936+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:20:40.940+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:20:40.940+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:20:40.949+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:20:40.947+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:20:40.950+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:20:40.990+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.067 seconds
[2025-04-21T21:21:11.316+0000] {processor.py:186} INFO - Started process (PID=400) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:21:11.317+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:21:11.321+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:21:11.321+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:21:11.329+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:21:11.327+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:21:11.330+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:21:11.353+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.047 seconds
[2025-04-21T21:21:41.531+0000] {processor.py:186} INFO - Started process (PID=420) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:21:41.534+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:21:41.541+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:21:41.540+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:21:41.550+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:21:41.548+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:21:41.550+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:21:41.588+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.069 seconds
[2025-04-21T21:22:12.078+0000] {processor.py:186} INFO - Started process (PID=440) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:22:12.079+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:22:12.082+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:22:12.081+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:22:12.090+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:22:12.088+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:22:12.090+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:22:12.121+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.056 seconds
[2025-04-21T21:22:42.340+0000] {processor.py:186} INFO - Started process (PID=460) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:22:42.341+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:22:42.344+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:22:42.343+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:22:42.349+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:22:42.347+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:22:42.349+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:22:42.369+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.037 seconds
[2025-04-21T21:23:12.795+0000] {processor.py:186} INFO - Started process (PID=480) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:23:12.796+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:23:12.806+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:23:12.805+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:23:12.818+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:23:12.816+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:23:12.818+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:23:12.852+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.067 seconds
[2025-04-21T21:23:43.029+0000] {processor.py:186} INFO - Started process (PID=1013) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:23:43.030+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:23:43.031+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:23:43.031+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:23:43.036+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:23:43.035+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:23:43.036+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:23:43.050+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.027 seconds
[2025-04-21T21:24:13.288+0000] {processor.py:186} INFO - Started process (PID=3529) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:24:13.289+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:24:13.292+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:24:13.291+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:24:13.307+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:24:13.304+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:24:13.308+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:24:13.326+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.050 seconds
[2025-04-21T21:24:43.562+0000] {processor.py:186} INFO - Started process (PID=6065) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:24:43.563+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:24:43.564+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:24:43.564+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:24:43.568+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:24:43.567+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:24:43.568+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:24:43.581+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.023 seconds
[2025-04-21T21:25:13.740+0000] {processor.py:186} INFO - Started process (PID=8366) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:25:13.740+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:25:13.743+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:25:13.743+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:25:13.749+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:25:13.747+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:25:13.749+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:25:13.773+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.039 seconds
[2025-04-21T21:25:44.285+0000] {processor.py:186} INFO - Started process (PID=10813) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:25:44.286+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:25:44.288+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:25:44.288+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:25:44.295+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:25:44.293+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:25:44.296+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:25:44.315+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.039 seconds
[2025-04-21T21:26:14.632+0000] {processor.py:186} INFO - Started process (PID=13310) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:26:14.633+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:26:14.636+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:26:14.636+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:26:14.640+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:26:14.639+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:26:14.641+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:26:14.661+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.035 seconds
[2025-04-21T21:26:44.988+0000] {processor.py:186} INFO - Started process (PID=15817) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:26:44.989+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:26:44.993+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:26:44.993+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:26:44.998+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:26:44.997+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:26:44.998+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:26:45.024+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.045 seconds
[2025-04-21T21:27:15.710+0000] {processor.py:186} INFO - Started process (PID=18121) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:27:15.712+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:27:15.718+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:27:15.717+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:27:15.724+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:27:15.722+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:27:15.724+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:27:15.738+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.037 seconds
[2025-04-21T21:27:46.012+0000] {processor.py:186} INFO - Started process (PID=19627) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:27:46.014+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:27:46.018+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:27:46.017+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:27:46.024+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:27:46.022+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:27:46.025+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:27:46.054+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.052 seconds
[2025-04-21T21:28:16.352+0000] {processor.py:186} INFO - Started process (PID=19647) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:28:16.354+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:28:16.360+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:28:16.359+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:28:16.368+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:28:16.366+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:28:16.369+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:28:16.397+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.059 seconds
[2025-04-21T21:34:41.096+0000] {processor.py:186} INFO - Started process (PID=169) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:34:41.097+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:34:41.100+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:34:41.100+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:34:41.105+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:34:41.104+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:34:41.106+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:34:41.131+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.038 seconds
[2025-04-21T21:35:11.250+0000] {processor.py:186} INFO - Started process (PID=186) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:35:11.251+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:35:11.255+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:35:11.255+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:35:11.264+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:35:11.262+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:35:11.264+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:35:11.287+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.058 seconds
[2025-04-21T21:35:41.648+0000] {processor.py:186} INFO - Started process (PID=214) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:35:41.651+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:35:41.655+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:35:41.655+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:35:41.662+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:35:41.661+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:35:41.663+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:35:41.690+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.052 seconds
[2025-04-21T21:36:12.093+0000] {processor.py:186} INFO - Started process (PID=2420) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:36:12.094+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:36:12.097+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:36:12.096+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:36:12.102+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:36:12.100+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:36:12.102+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:36:12.122+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.037 seconds
[2025-04-21T21:36:42.497+0000] {processor.py:186} INFO - Started process (PID=4195) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:36:42.498+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:36:42.502+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:36:42.502+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:36:42.511+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:36:42.507+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:36:42.512+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:36:42.544+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.062 seconds
[2025-04-21T21:37:13.195+0000] {processor.py:186} INFO - Started process (PID=4215) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:37:13.196+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:37:13.200+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:37:13.200+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:37:13.212+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:37:13.209+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:37:13.213+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:37:13.253+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.076 seconds
[2025-04-21T21:37:43.918+0000] {processor.py:186} INFO - Started process (PID=4235) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:37:43.929+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:37:43.938+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:37:43.936+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:37:43.986+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:37:43.979+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:37:43.987+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:37:44.024+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.134 seconds
[2025-04-21T21:38:14.480+0000] {processor.py:186} INFO - Started process (PID=4255) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:38:14.482+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:38:14.491+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:38:14.490+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:38:14.498+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:38:14.496+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:38:14.499+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:38:14.536+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.071 seconds
[2025-04-21T21:38:44.931+0000] {processor.py:186} INFO - Started process (PID=4275) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:38:44.933+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:38:44.942+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:38:44.941+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:38:44.953+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:38:44.951+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:38:44.953+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:38:44.993+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.075 seconds
[2025-04-21T21:39:15.125+0000] {processor.py:186} INFO - Started process (PID=4295) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:39:15.129+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:39:15.133+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:39:15.133+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:39:15.142+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:39:15.140+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:39:15.143+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:39:15.181+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.083 seconds
[2025-04-21T21:39:45.546+0000] {processor.py:186} INFO - Started process (PID=4315) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:39:45.548+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:39:45.552+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:39:45.551+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:39:45.561+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:39:45.556+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:39:45.562+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:39:45.593+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.060 seconds
[2025-04-21T21:40:16.026+0000] {processor.py:186} INFO - Started process (PID=4335) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:40:16.029+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:40:16.038+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:40:16.038+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:40:16.047+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:40:16.045+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:40:16.047+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:40:16.076+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.066 seconds
[2025-04-21T21:40:46.492+0000] {processor.py:186} INFO - Started process (PID=4355) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:40:46.493+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:40:46.501+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:40:46.501+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:40:46.512+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:40:46.509+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:40:46.513+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:40:46.543+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.061 seconds
[2025-04-21T21:41:16.978+0000] {processor.py:186} INFO - Started process (PID=4375) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:41:16.979+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:41:16.984+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:41:16.983+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:41:16.991+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:41:16.990+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:41:16.992+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:41:17.022+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.058 seconds
[2025-04-21T21:41:47.483+0000] {processor.py:186} INFO - Started process (PID=4395) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:41:47.484+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:41:47.492+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:41:47.489+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:41:47.505+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:41:47.503+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:41:47.505+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:41:47.529+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.062 seconds
[2025-04-21T21:42:18.017+0000] {processor.py:186} INFO - Started process (PID=4415) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:42:18.020+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:42:18.031+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:42:18.030+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:42:18.039+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:42:18.037+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:42:18.039+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:42:18.070+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.070 seconds
[2025-04-21T21:42:48.540+0000] {processor.py:186} INFO - Started process (PID=4435) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:42:48.542+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:42:48.554+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:42:48.550+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:42:48.564+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:42:48.562+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:42:48.565+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:42:48.589+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.066 seconds
[2025-04-21T21:43:18.815+0000] {processor.py:186} INFO - Started process (PID=4455) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:43:18.816+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:43:18.825+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:43:18.825+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:43:18.833+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:43:18.831+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:43:18.833+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:43:18.865+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.066 seconds
[2025-04-21T21:43:49.393+0000] {processor.py:186} INFO - Started process (PID=4475) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:43:49.395+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:43:49.409+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:43:49.406+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:43:49.418+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:43:49.417+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:43:49.419+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:43:49.445+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.063 seconds
[2025-04-21T21:44:19.949+0000] {processor.py:186} INFO - Started process (PID=4495) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:44:19.951+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:44:19.955+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:44:19.954+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:44:19.962+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:44:19.960+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:44:19.962+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:44:19.985+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.044 seconds
[2025-04-21T21:44:50.538+0000] {processor.py:186} INFO - Started process (PID=4515) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:44:50.540+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:44:50.543+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:44:50.543+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:44:50.550+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:44:50.548+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:44:50.550+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:44:50.575+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.049 seconds
[2025-04-21T21:45:21.092+0000] {processor.py:186} INFO - Started process (PID=4535) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:45:21.094+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:45:21.097+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:45:21.097+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:45:21.103+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:45:21.101+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:45:21.103+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:45:21.120+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.052 seconds
[2025-04-21T21:45:51.564+0000] {processor.py:186} INFO - Started process (PID=4555) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:45:51.565+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:45:51.568+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:45:51.568+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:45:51.574+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:45:51.572+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:45:51.575+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:45:51.592+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.036 seconds
[2025-04-21T21:46:22.153+0000] {processor.py:186} INFO - Started process (PID=4575) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:46:22.156+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:46:22.157+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:46:22.157+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:46:22.162+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:46:22.160+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:46:22.162+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:46:22.176+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.029 seconds
[2025-04-21T21:46:52.383+0000] {processor.py:186} INFO - Started process (PID=4595) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:46:52.384+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:46:52.387+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:46:52.386+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:46:52.392+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:46:52.391+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:46:52.393+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:46:52.411+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.036 seconds
[2025-04-21T21:47:22.957+0000] {processor.py:186} INFO - Started process (PID=4615) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:47:22.959+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:47:22.961+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:47:22.961+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:47:22.966+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:47:22.965+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:47:22.967+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:47:22.985+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.038 seconds
[2025-04-21T21:47:53.523+0000] {processor.py:186} INFO - Started process (PID=4635) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:47:53.524+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:47:53.525+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:47:53.525+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:47:53.531+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:47:53.529+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:47:53.532+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:47:53.549+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.030 seconds
[2025-04-21T21:48:24.149+0000] {processor.py:186} INFO - Started process (PID=4655) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:48:24.150+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:48:24.152+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:48:24.152+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:48:24.155+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:48:24.154+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:48:24.155+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:48:24.168+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.024 seconds
[2025-04-21T21:48:54.564+0000] {processor.py:186} INFO - Started process (PID=4675) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:48:54.565+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:48:54.566+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:48:54.566+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:48:54.570+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:48:54.569+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:48:54.570+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:48:54.582+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.026 seconds
[2025-04-21T21:49:25.176+0000] {processor.py:186} INFO - Started process (PID=4695) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:49:25.177+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:49:25.178+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:49:25.178+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:49:25.181+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:49:25.180+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:49:25.181+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:49:25.193+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.021 seconds
[2025-04-21T21:49:55.821+0000] {processor.py:186} INFO - Started process (PID=4713) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:49:55.823+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:49:55.824+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:49:55.824+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:49:55.827+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:49:55.827+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:49:55.828+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:49:55.838+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.022 seconds
[2025-04-21T21:50:26.496+0000] {processor.py:186} INFO - Started process (PID=4722) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:50:26.498+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:50:26.505+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:50:26.504+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:50:26.516+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:50:26.514+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:50:26.517+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:50:26.554+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.081 seconds
[2025-04-21T21:50:57.060+0000] {processor.py:186} INFO - Started process (PID=4740) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:50:57.061+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:50:57.065+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:50:57.064+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:50:57.072+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:50:57.070+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:50:57.072+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:50:57.097+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.045 seconds
[2025-04-21T21:51:27.726+0000] {processor.py:186} INFO - Started process (PID=4760) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:51:27.728+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:51:27.734+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:51:27.732+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:51:27.744+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:51:27.742+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:51:27.744+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:51:27.780+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.070 seconds
[2025-04-21T21:51:58.481+0000] {processor.py:186} INFO - Started process (PID=4780) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:51:58.483+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:51:58.488+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:51:58.487+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:51:58.498+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:51:58.496+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:51:58.499+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:51:58.529+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.066 seconds
[2025-04-21T21:52:28.690+0000] {processor.py:186} INFO - Started process (PID=4800) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:52:28.695+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:52:28.704+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:52:28.703+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:52:28.712+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:52:28.710+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:52:28.713+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:52:28.748+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.075 seconds
[2025-04-21T21:52:59.175+0000] {processor.py:186} INFO - Started process (PID=4820) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:52:59.180+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:52:59.186+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:52:59.185+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:52:59.194+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:52:59.192+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:52:59.194+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:52:59.230+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.069 seconds
[2025-04-21T21:53:29.917+0000] {processor.py:186} INFO - Started process (PID=4840) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:53:29.919+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:53:29.924+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:53:29.923+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:53:29.931+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:53:29.930+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:53:29.932+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:53:29.967+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.074 seconds
[2025-04-21T21:54:00.518+0000] {processor.py:186} INFO - Started process (PID=4860) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:54:00.520+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:54:00.525+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:54:00.525+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:54:00.533+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:54:00.531+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:54:00.534+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:54:00.567+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.068 seconds
[2025-04-21T21:54:31.045+0000] {processor.py:186} INFO - Started process (PID=4880) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:54:31.051+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:54:31.056+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:54:31.055+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:54:31.063+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:54:31.061+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:54:31.063+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:54:31.094+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.072 seconds
[2025-04-21T21:55:01.247+0000] {processor.py:186} INFO - Started process (PID=4900) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:55:01.256+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:55:01.260+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:55:01.260+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:55:01.266+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:55:01.265+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:55:01.266+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:55:01.300+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.072 seconds
[2025-04-21T21:55:31.774+0000] {processor.py:186} INFO - Started process (PID=4920) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:55:31.779+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:55:31.784+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:55:31.783+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:55:31.791+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:55:31.790+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:55:31.792+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:55:31.826+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.066 seconds
[2025-04-21T21:56:02.284+0000] {processor.py:186} INFO - Started process (PID=4940) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:56:02.286+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:56:02.291+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:56:02.290+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:56:02.314+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:56:02.312+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:56:02.314+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:56:02.340+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.065 seconds
[2025-04-21T21:56:32.780+0000] {processor.py:186} INFO - Started process (PID=4960) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:56:32.783+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:56:32.788+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:56:32.787+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:56:32.798+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:56:32.796+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:56:32.798+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:56:32.834+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.071 seconds
[2025-04-21T21:57:03.355+0000] {processor.py:186} INFO - Started process (PID=4980) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:57:03.357+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:57:03.363+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:57:03.363+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:57:03.376+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:57:03.374+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:57:03.376+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:57:03.411+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.073 seconds
[2025-04-21T21:57:33.833+0000] {processor.py:186} INFO - Started process (PID=5000) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:57:33.836+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:57:33.839+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:57:33.839+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:57:33.850+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:57:33.847+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:57:33.850+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:57:33.881+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.063 seconds
[2025-04-21T21:58:04.296+0000] {processor.py:186} INFO - Started process (PID=5020) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:58:04.298+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:58:04.302+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:58:04.302+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:58:04.310+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:58:04.308+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:58:04.310+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:58:04.346+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.071 seconds
[2025-04-21T21:58:34.849+0000] {processor.py:186} INFO - Started process (PID=5040) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:58:34.850+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:58:34.856+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:58:34.855+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:58:34.864+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:58:34.862+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:58:34.864+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:58:34.899+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.063 seconds
[2025-04-21T21:59:05.365+0000] {processor.py:186} INFO - Started process (PID=5060) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:59:05.366+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:59:05.370+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:59:05.369+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:59:05.383+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:59:05.381+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:59:05.384+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:59:05.409+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.055 seconds
[2025-04-21T21:59:35.970+0000] {processor.py:186} INFO - Started process (PID=5080) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:59:35.973+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T21:59:35.977+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:59:35.977+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:59:35.991+0000] {logging_mixin.py:190} INFO - [2025-04-21T21:59:35.989+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T21:59:35.993+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T21:59:36.020+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.066 seconds
[2025-04-21T22:00:06.400+0000] {processor.py:186} INFO - Started process (PID=5100) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T22:00:06.402+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T22:00:06.405+0000] {logging_mixin.py:190} INFO - [2025-04-21T22:00:06.404+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T22:00:06.413+0000] {logging_mixin.py:190} INFO - [2025-04-21T22:00:06.410+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T22:00:06.413+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T22:00:06.440+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.050 seconds
[2025-04-21T22:00:36.932+0000] {processor.py:186} INFO - Started process (PID=5120) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T22:00:36.932+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T22:00:36.936+0000] {logging_mixin.py:190} INFO - [2025-04-21T22:00:36.935+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T22:00:36.941+0000] {logging_mixin.py:190} INFO - [2025-04-21T22:00:36.940+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T22:00:36.941+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T22:00:36.956+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.033 seconds
[2025-04-21T22:01:07.716+0000] {processor.py:186} INFO - Started process (PID=5140) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T22:01:07.718+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T22:01:07.725+0000] {logging_mixin.py:190} INFO - [2025-04-21T22:01:07.724+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T22:01:07.736+0000] {logging_mixin.py:190} INFO - [2025-04-21T22:01:07.734+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T22:01:07.736+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T22:01:07.771+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.070 seconds
[2025-04-21T22:01:38.449+0000] {processor.py:186} INFO - Started process (PID=5160) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T22:01:38.451+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T22:01:38.459+0000] {logging_mixin.py:190} INFO - [2025-04-21T22:01:38.459+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T22:01:38.477+0000] {logging_mixin.py:190} INFO - [2025-04-21T22:01:38.471+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T22:01:38.477+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T22:01:38.530+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.113 seconds
[2025-04-21T22:02:08.954+0000] {processor.py:186} INFO - Started process (PID=5180) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T22:02:08.955+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T22:02:08.958+0000] {logging_mixin.py:190} INFO - [2025-04-21T22:02:08.958+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T22:02:08.967+0000] {logging_mixin.py:190} INFO - [2025-04-21T22:02:08.965+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T22:02:08.967+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T22:02:08.995+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.050 seconds
[2025-04-21T22:02:39.128+0000] {processor.py:186} INFO - Started process (PID=5200) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T22:02:39.130+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-21T22:02:39.138+0000] {logging_mixin.py:190} INFO - [2025-04-21T22:02:39.138+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T22:02:39.154+0000] {logging_mixin.py:190} INFO - [2025-04-21T22:02:39.146+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-21T22:02:39.155+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-21T22:02:39.197+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.086 seconds
